sql-plugin/src/main/spark332cdh/scala/com/nvidia/spark/rapids/shims/spark332cdh/SparkShimServiceProvider.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION.
```

Comment:
remove 2022 from copyright since new file

Reply:
Done. Fixed in latest commit.